### ISSUE TITLE
PlaceType enum is missing mappings

### ISSUE THREAD

**Issue by ChasakisD** [xamarin/GoogleApisForiOSComponents#354]
Version 3.5.0

App Crashes and there is no way to catch that exception. Please sync the library with the Google Places API.

Exception: tourist_attraction has no associated enum value in PlaceType on this platform.

```
Unhandled Exception:
System.NotSupportedException: tourist_attraction has no associated enum value in PlaceType on this platform.
  at Google.Places.PlaceTypeExtensions.GetValue (Foundation.NSString constant) [0x0082e] in <f93e8174f2244446be6a465f88ca7944>:0 
  at Google.Places.AutocompletePrediction+<>c.<get_Types>b__14_0 (System.IntPtr ptr) [0x00007] in <f93e8174f2244446be6a465f88ca7944>:0 
  at Foundation.NSArray.ArrayFromHandleFunc[T] (System.IntPtr handle, System.Func`2[T,TResult] createObject) [0x00027] in /Library/Frameworks/Xamarin.iOS.framework/Versions/13.8.3.0/src/Xamarin.iOS/Foundation/NSArray.cs:353 
  at Google.Places.AutocompletePrediction.get_Types () [0x0002b] in <f93e8174f2244446be6a465f88ca7944>:0 
  at MyApp.Xamarin.Google.Places.iOS.GoogleMapsPlacesService+<>c.<GetLocationPredictionsAsync>b__2_1 (Google.Places.AutocompletePrediction p) [0x00022] in <ac770904c18c41fa99420d1b66186960>:0 
  at System.Linq.Enumerable+SelectArrayIterator`2[TSource,TResult].ToList () [0x00014] in /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/src/Xamarin.iOS/external/corefx/src/System.Linq/src/System/Linq/Select.cs:252 
  at System.Linq.Enumerable.ToList[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x0000e] in /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/src/Xamarin.iOS/external/corefx/src/System.Linq/src/System/Linq/ToCollection.cs:30 
```

Exception2: supermarket has no associated enum value in PlaceType on this platform.

```
Unhandled Exception:
System.NotSupportedException: supermarket has no associated enum value in PlaceType on this platform.
  at Google.Places.PlaceTypeExtensions.GetValue (Foundation.NSString constant) [0x0082e] in <f93e8174f2244446be6a465f88ca7944>:0 
  at Google.Places.AutocompletePrediction+<>c.<get_Types>b__14_0 (System.IntPtr ptr) [0x00007] in <f93e8174f2244446be6a465f88ca7944>:0 
  at Foundation.NSArray.ArrayFromHandleFunc[T] (System.IntPtr handle, System.Func`2[T,TResult] createObject) [0x00027] in /Library/Frameworks/Xamarin.iOS.framework/Versions/13.8.3.0/src/Xamarin.iOS/Foundation/NSArray.cs:353 
  at Google.Places.AutocompletePrediction.get_Types () [0x0002b] in <f93e8174f2244446be6a465f88ca7944>:0 
  at MyApp.Xamarin.Google.Places.iOS.GoogleMapsPlacesService+<>c.<GetLocationPredictionsAsync>b__2_1 (Google.Places.AutocompletePrediction p) [0x00022] in <ac770904c18c41fa99420d1b66186960>:0 
  at System.Linq.Enumerable+SelectArrayIterator`2[TSource,TResult].ToList () [0x00014] in /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/src/Xamarin.iOS/external/corefx/src/System.Linq/src/System/Linq/Select.cs:252 
  at System.Linq.Enumerable.ToList[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x0000e] in /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/src/Xamarin.iOS/external/corefx/src/System.Linq/src/System/Linq/ToCollection.cs:30 
```

Exception3: continent has no associated enum value in PlaceType on this platform.

```
Unhandled Exception:
System.NotSupportedException: continent has no associated enum value in PlaceType on this platform. (System.NotSupportedException)
  at Google.Places.PlaceTypeExtensions.GetValue (Foundation.NSString constant) [0x0082e] in <f93e8174f2244446be6a465f88ca7944>:0 
  at Google.Places.AutocompletePrediction+<>c.<get_Types>b__14_0 (System.IntPtr ptr) [0x00007] in <f93e8174f2244446be6a465f88ca7944>:0 
  at Foundation.NSArray.ArrayFromHandleFunc[T] (System.IntPtr handle, System.Func`2[T,TResult] createObject) [0x00027] in /Library/Frameworks/Xamarin.iOS.framework/Versions/13.8.3.0/src/Xamarin.iOS/Foundation/NSArray.cs:353 
  at Google.Places.AutocompletePrediction.get_Types () [0x0002b] in <f93e8174f2244446be6a465f88ca7944>:0 
  at MyApp.Xamarin.Google.Places.iOS.GoogleMapsPlacesService+<>c.<GetLocationPredictionsAsync>b__2_1 (Google.Places.AutocompletePrediction p) [0x00022] in <ac770904c18c41fa99420d1b66186960>:0 
  at System.Linq.Enumerable+SelectArrayIterator`2[TSource,TResult].ToList () [0x00014] in /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/src/Xamarin.iOS/external/corefx/src/System.Linq/src/System/Linq/Select.cs:252 
  at System.Linq.Enumerable.ToList[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x0000e] in /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/src/Xamarin.iOS/external/corefx/src/System.Linq/src/System/Linq/ToCollection.cs:30

**Comment by SotoiGhost** [comment]
@ChasakisD We have released Google Places v3.7.0. Please, update and let us know your results. Hope this helps you! Best!

**Comment by Redth** [comment]
We are closing this issue since no response was received.  Please open a new issue referring to this one if you have more information.
